tabsRecursion.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tab Recursion</title>
+</head>
+<body>
+        <button onclick="openTab()">open tab</button>
+        <button onclick="closeTab()">close tab</button>
+</body>
+</html>
+
+<script>
+
+    var windows = [];
```

Comment:
Prefer `const` over `let` and `let` over `var`

if we are not modifying variable value then declare it as `const`